concourse/tasks/remove-healthcheck-db.yml

```diff
@@ -13,11 +13,17 @@ run:
     - |
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
       . ./config/config.sh
-      if curl -I -f $API_ENDPOINT/info; then
-        echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-        cf target -o testers -s healthcheck
-        cf delete healthcheck -f -r
-        if cf services | grep -q healthcheck-db; then
-          cf delete-service healthcheck-db -f
-        fi
+      if ! curl -I -f $API_ENDPOINT/info; then
+        echo "CF API unavailable. Skipping..."
+        exit 0
+      fi
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+      if ! cf org testers > /dev/null; then
```

Comment:
Ok, that's right. At the other hand I checked if we can just delete the org but it starts deleting the service instance and fails with "An operation for service instance test is in progress" and doesn't delete the app.  So I have just small improvement suggestion and we should be good to go.

Can we replace
```
if ! curl -I -f $API_ENDPOINT/info; then
```

with 

```
if echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS} ; then
```
and remove the login command below so we'll have api check and login in one line.